### PR TITLE
Update docs [skip ci]

### PR DESCRIPTION
**Pull request by pana1990**
1º path test update in 4e8ed67.
2º language options works if use double quotes or without quotes but it do not work with single quotes.

### CODE REVIEW

File: docs/guide/basic-usage.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Testing
 -----------
 
 Define a `tests` alias in your console config. For example, for the `basic` project template, this should be added
-to the `console.php` configuration: `Yii::setAlias('tests', __DIR__ . '/../tests');`
+to the `console.php` configuration: `Yii::setAlias('tests', __DIR__ . '/tests/codeception');`
```
Comment (cebe) [inline docs/guide/basic-usage.md]: if this code goes into [console.php](https://github.com/yiisoft/yii2-app-basic/blob/a2b4a8677c27d7a762d4432c237f2034a3b1b4ba/config/console.php) is should go one directory upwards. Should be `/../tests/codeception`

Comment (pana1990) [inline docs/guide/basic-usage.md]: currenly it is ''https://github.com/yiisoft/yii2-app-basic/blob/a2b4a8677c27d7a762d4432c237f2034a3b1b4ba/config/console.php#L3"

Comment (cebe) [inline docs/guide/basic-usage.md]: note that there is `dirname(__DIR__)` which is basically `__DIR__ . '/../'`. so adding dirname here would be fine too.

Comment (pana1990) [inline docs/guide/basic-usage.md]: ups, it's truth, sorry, i will correct my mistake.

Comment (pana1990) [inline docs/guide/basic-usage.md]: @cebe done!

**Comment by cebe** [comment]
Thanks!

**Pull request merged**